contrib/skkdict/template.py

```diff
@@ -0,0 +1,17 @@
+pkgname = "skkdict"
+pkgver = "1_git20231226"
```

Comment:
```suggestion
pkgver = "0_git20231226"
```

stuff with no version starts at 0 for this (and if there's a 'latest' version, then it would be that version with _git appended, etc)